core/src/main/java/org/elasticsearch/index/SearchSlowLog.java

```diff
@@ -73,20 +73,17 @@
     public static final Setting<TimeValue> INDEX_SEARCH_SLOWLOG_THRESHOLD_FETCH_TRACE_SETTING =
         Setting.timeSetting(INDEX_SEARCH_SLOWLOG_PREFIX + ".threshold.fetch.trace", TimeValue.timeValueNanos(-1),
             TimeValue.timeValueMillis(-1), Property.Dynamic, Property.IndexScope);
-    public static final Setting<Boolean> INDEX_SEARCH_SLOWLOG_REFORMAT =
-        Setting.boolSetting(INDEX_SEARCH_SLOWLOG_PREFIX + ".reformat", true, Property.Dynamic, Property.IndexScope);
```

Comment:
let me guess, we still had the setting but it had no effect? We were reading and passing it all the way to the log printer but we didn't check the flag value when needed? That is why also removing this setting should not be considered a breaking change, as the setting didn't have any effect?